content/docs/command-reference/dag.md

```diff
@@ -6,7 +6,8 @@ connected [stages](/doc/command-reference/run).
 ## Synopsis
 
 ```usage
-usage: dvc dag [-h] [-q | -v] [--dot] [--full] [target]
+usage: dvc dag [-h] [-q | -v] [--dot] [--mermaid] [--md]
+               [--full] [target]
```

Comment:
Is this the order in which they're printed by the tool?

> I'm on DVC 2.10.1 and don't see this usage output yet.

Reply:
Well in any case let's just decide the sorting (and grouping) here according to most-useful (we can update the core repo after, to reflect this sorting). I propose:

```suggestion
usage: dvc dag [-h] [-q | -v] [--full]
               [--md] [--mermaid] [--dot]
               [target]
```

Reply:
> Is this the order in which they're printed by the tool?

```
usage: dvc dag [-h] [-q | -v] [--dot] [--mermaid] [--md] [--full] [-o]
               [target]
```